spring-cloud-skipper-server-core/src/main/resources/application.yml

```diff
@@ -36,7 +36,12 @@ spring:
               checksum-sha1-url: "{repository}/org/springframework/cloud/spring-cloud-skipper-shell/{version}/spring-cloud-skipper-shell-{version}.jar.sha1"
               checksum-sha256-url: "{repository}/org/springframework/cloud/spring-cloud-skipper-shell/{version}/spring-cloud-skipper-shell-{version}.jar.sha256"
         package-repositories:
-          local:
+          -
```

Comment:
We removed this on [purpose](https://github.com/spring-cloud/spring-cloud-skipper/commit/d7f9baf615bc801b7e4cb38e77421653d0848cc0#diff-d23f8c4327c765ff9329c887fb12b5ec): https://github.com/spring-cloud/spring-cloud-skipper/issues/762